### PR TITLE
adds a try catch to prevent permission exception

### PR DESCRIPTION
**Pull request by HGotthard**
## Requirements for Contributing

* [x] The changed and/or added Code should be documented and understandable.
* [x] The changed and/or added Code should follow the Code Conventions.

## Type of the Pull Request
* [x] Bugfix
* [ ] Feature change
* [ ] Documentation change
* [ ] Something different

## Description of the Changes
Adds a try catch to the kick and ban command to catch the InsufficientPermissionException and sends an error message.

### CODE REVIEW

**Comment by ShowMeYourSkil** [comment]
The guild with the problem has database entries from users who do not exist or are not on the guild. I will manually delete the mutes from the guild.

**Comment by ShowMeYourSkil** [comment]
ready to merge

File: src/main/java/de/progen_bot/commands/moderator/blacklist/CommandBan.java [hunk 1]
```diff
@@ -59,34 +54,36 @@ public void execute(CommandManager.ParsedCommandString parsedCommand, MessageRec
         if (eb == null)
             return;
 
-        if (parsedCommand.getArgs().length > 0)
-        {
-            User user = event.getJDA().retrieveUserById(event.getMessage().getMentionedUsers().get(0).getId()).complete();
+        if (parsedCommand.getArgs().length > 0) {
+            User user = event.getJDA().retrieveUserById(event.getMessage().getMentionedUsers().get(0).getId())
```
Comment (byNoobiYT) [inline src/main/java/de/progen_bot/commands/moderator/blacklist/CommandBan.java:58]: Could throw NPE or IndexOutOfBoundsException

**Review by byNoobiYT — APPROVED** [review]
fine

**Pull request merged**